resource/geo-density-chart.js

```diff
@@ -279,19 +279,17 @@ function GeoDensityChart(){
  * Format the tooltip to be displayed in HTML
  * tooltip = {"reviews_per_month": 0.92, "room_type": "Private Room"};
  * @param tooltip
- * @returns {number}
+ * @returns {string}
  */
 function formatTooltip(tooltip){
-    if (!tooltip){
-        return
+    if (jQuery.isEmptyObject(tooltip)) {
```

Comment:
Are you sure about using jQuery? Granted, this solution is cleaner than[ native js code](https://stackoverflow.com/questions/679915/how-do-i-test-for-an-empty-javascript-object) but jQuery has become quite slow nowadays and I heard that most frontenders advised against it

Reply:
I did not know that, thanks for pointing it out, I used an alternative in c5ef65a